### DISCUSSION TITLE
Large Nuclear reactor not working

### DISCUSSION THREAD

**Discussion by disepto** [HbmMods/Hbm-s-Nuclear-Tech-GIT#901] [General]
I'm not sure if this is an intentional change or a bug, but i looked through 10 pages of the change logs and couldn't find anything about changes with the depleted fuel thing. I used to play on the 1.12.2 version where you could just put empty fuel rods in the input slot and hot depleted rods would come out. In every 1.8 version i've tried to use, this does not work. I don't know how else I'm supposed to get depleted uranium or other things in survival without skipping directly to a ZIRNOX, and I hope this is fixed or clarified in future updates.

**Comment by SoB1nary** [comment]
First of all, you're not *skipping* to a zirnox, that thing is cheaper than a large reactor. Secondly, the fuel ejector exists: place it instead of an access hatch and add a crate nearby. I do think that the UI elements for rods should be removed completely, given that they do not work now.

**Reply by disepto** [reply]
THANK YOU!!!!
